testdata/sampleconfig/power-toggle-config.yaml

```diff
@@ -72,3 +72,6 @@ aws:
 
 # misc settings --------------------------------------------------------------------------------------------------------
 log_level: DEBUG
+# mock server for testing
```

Comment:
Can we add another option to this file that enables/disables experimental features. We can have this feature as experimental since there's still TODO items (like persistence) to figure out.

Reply:
also the mock should be disabled by default

Reply:
Agreed. Experimental flag is a good idea.